backend/elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/search/impl/ElasticsearchDslConverterHandler.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.search.impl;
+
+import org.hibernate.search.engine.search.predicate.DslConverter;
+
+public class ElasticsearchDslConverterHandler {
+
+	public void handle(DslConverter dslConverter) {
+		// do nothing
+	}
```

Comment:
As I mentioned, it's be better if this callback also returned the `ToDocumentFieldValueConverter` instance...

Anyway, I'll get back to you with a simpler solution (I hope).